x/dex/keeper/integration_cancellimitorder_test.go

```diff
@@ -249,7 +249,6 @@ func (s *DexTestSuite) TestCancelGoodTilAfterExpirationFails() {
 
 	// WHEN expiration date has passed
 	s.nextBlockWithTime(time.Now().AddDate(0, 0, 2))
-	// TODO: s.App.EndBlock(abci.RequestEndBlock{Height: 0})
```

Comment:
I remember you talked about inability to call endblock in Cosmos SDK 0.50. How did you manage to resolve that?

Reply:
it was about `TestPlaceLimitOrderGoodTilExpiresNotPurged` test. I needed to move chain forward without calling endblocker. But it turned out, it's enough just to change time in `ctx` to simulate that chain is alive.